addons/sale/models/sale.py

```diff
@@ -506,6 +506,33 @@ def action_view_invoice(self):
             action = {'type': 'ir.actions.act_window_close'}
         return action
 
+    @api.multi
+    def _finalize_invoices(self, invoices, references):
```

Comment:
please add a docstring, invoices is not a recordset but a dict formatted specifically so i'd rather it be clear in the doc

Reply:
What about call it `invoices_vals` or `invoice_vals_list`?

Reply:
Now seeing the code it should be more `invoice_groups` IMO

Reply:
I added the docstring. I kept the argument names, though as I found it easier to relate to how the variables are named in the calling method.

Reply:
OK, good enough